.circleci/config.yml

```diff
@@ -53,4 +75,96 @@ jobs:
           path: ./test/rendering/__artifacts__
       - store_test_results:
           path: coverage/junit
-      - save_cache: *save_yarn_cache
+      - persist_to_workspace:
+          root: ~/action-button
+          paths:
+            - dist
+            - package.json
+            - core
+
+  # publish-dev:
```

Comment:
I've never used this, but we have it for several charts. I guess it can be removed?

Reply:
I guess you can use it to test your extension? Is it commented in all other charts as well? In that case better to remove I think

Reply:
No I commented it now